lms/urls.py

```diff
@@ -32,12 +32,12 @@
     url(r'^login_ajax/(?P<error>[^/]*)$', 'student.views.login_user'),
 
     url(r'^email_confirm/(?P<key>[^/]*)$', 'student.views.confirm_email_change'),
-    url(r'^event$', 'track.views.user_track'),
-    url(r'^performance$', 'performance.views.performance_log'),
-    url(r'^segmentio/event$', 'track.views.segmentio.segmentio_event'),
 
-    # TODO: Is this used anymore? What is STATIC_GRAB?
-    url(r'^t/(?P<template>[^/]*)$', 'static_template_view.views.index'),
+    url(r'', include('track.urls')),
+
+    url(r'', include('performance.urls')),
```

Comment:
Nit: this is very nit-picky, but I like the convention of having a comment before each grouping of URLs to describe what it is doing. If some of these are self-documenting, I probably would group them together and not have an extra line between each one. I would also recommend keeping them in alphabetical order so that they can be found straightforwardly.